apps/verticalmessage/vertical_message.star

```diff
@@ -81,6 +89,20 @@ def get_schema():
                 default = "tb-8",
                 options = fonts,
             ),
+            schema.Text(
+                id = "linespacing",
+                name = "Line Spacing",
```

Comment:
We really, really need a number schema field 😭

Reply:
I would appreciate it yes :)

You'll then be getting a ton of PRs though